### PR TITLE
Add To do list project

### PR DESCRIPTION
**Pull request by Reem-lab**
## Changes that I made here 👍 
 - set up the project with my webpack template 
 - Create the Tasks dynamically by input 
 - Add remove trash to remove tasks
 - Add completed and index to the task
 - save them in local storage
 - when you delete the item from the list it deleted from local storage also
 - no linter error
 - Add remove all completed functionality
 - Add function to update status from completed false to true
 - Add some styles to my list

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+function getDataFromLocal() {
+  const data = window.localStorage.getItem('tasks');
+  if (data) {
+    const tasks = JSON.parse(data); // convert into an object
+    addElementsToPageFrom(tasks);
+  }
+}
+
+function deleteTaskWith(taskId) {
+  // filter //taskId = e.target.parentelemnt.getAttribute(data-id);
```
Comment (gabyse1) [inline src/index.js]: I kindly ask you to remove this commented line to make your code cleaner.

**Review by gabyse1 — COMMENTED** [review]
# Amazing job @Reem-lab 
First of all, congratulations on your great job!!! 👏👏👏

## Highlights ✔️
- Your project's features work properly (Adding, updating and deleting tasks).
- There are no linter erros.
- You have a DRY, KISS, and YAGNI code, it looks very nice.
- Descriptive pull request title.
- Complete pull request description.
- Good Readme.md file description.

## Changes Required  ♻️ 🙂
- Only there is an unused commented code to remove from your index.js file. I kindly ask you to drop it. It will make your code cleaner and readable.
*Please check the comments under the review*

#### Good luck and Happy coding!!! 👏💻

*Feel free to leave any questions or comments in the PR thread if something is not 💯 clear.*

File: src/index.js (hunk 1 above)
Comment (Reem-lab) [reply src/index.js]: Thanks for this review

**Review by gabyse1 — APPROVED** [review]
# Status: Approved!!!  🥇🎉

## Hi @Reem-lab 👋,
## Your project is complete. 🥳
There's nothing to say other than your project is ready to be merged whenever you see fit! 😊
#### Happy coding during the next milestone.!!! 👏💻

**Pull request merged**